MKLDNN_README.md

```diff
@@ -0,0 +1,287 @@
+# Build/Install MXNet with MKL-DNN
+
+<h2 id="0">Contents</h2>
+
+* [1. Linux](#1)
+* [2. MacOS](#2)
+* [3. Windows](#3)
+* [4. Verify MXNet with python](#4)
+* [5. Enable MKL BLAS](#5)
+
+<h2 id="1">Linux</h2>
+
+### Prerequisites
+
+```
+apt-get update && apt-get install -y build-essential git libopencv-dev curl gcc libopenblas-dev python python-pip python-dev python-opencv graphviz python-scipy python-sklearn
+```
+
+### Clone MXNet sources
+
+```
+git clone --recursive https://github.com/apache/incubator-mxnet.git
+cd incubator-mxnet
+git submodule update --recursive --init
+```
+
+### Build MXNet with MKL-DNN
+
+```
+make -j $(nproc) USE_OPENCV=1 USE_MKLDNN=1 USE_BLAS=mkl USE_INTEL_PATH=/opt/intel
+```
+
+If you don't have full MKL library installed, you can use OpenBLAS by setting `USE_BLAS=openblas`.

```

Comment:
Can you clarify this state? Why wouldn't the user have the full MKL lib? That's not in the prerequisites? Can you link to installation instructions, so the user can upgrade to the full install? What happens if you don't have it?

Reply:
keep consistent with https://mxnet.incubator.apache.org/install/windows_setup.html

Reply:
MKL is a faster implementation of openblas. one can replace the other.